packages/grid/src/Grid.tsx

```diff
@@ -791,15 +806,19 @@ class Grid extends PureComponent<GridProps, GridState> {
   }
 
   updateCanvas(metrics = this.updateMetrics()): void {
+    if (!this.canvasContext) throw new Error('canvasContext not set');
+
```

Comment:
You don't need this - the second line in `updateCanvasScale()` already does this.
```suggestion
```